tyndale-bulletin.csl

```diff
@@ -267,6 +265,19 @@
       <else-if type="bill book graphic legal_case legislation motion_picture report song" match="any">
         <text variable="title" font-style="italic" text-case="title"/>
       </else-if>
+      <!-- Include "review of " prefix, italicize the book titles in untitled book reviews per §§11.3.8-11.3.9. -->
+      <else-if variable="reviewed-author">
+        <choose>
+          <!-- Include the review title, if avaialable, in quotation marks per §11.3.8. -->
+          <if variable="reviewed-title">
+            <text variable="title" quotes="true" suffix=", " text-case="title"/>
+            <text variable="reviewed-title" font-style="italic" prefix="review of " text-case="title"/>
```

Comment:
```suggestion
            <group delimiter=", ">
              <text variable="title" quotes="true" suffix=", " text-case="title"/>
              <group delimiter=" ">
                <text term="review-of" font-style="italic"/>
                <text variable="reviewed-title" font-style="italic" text-case="title"/>
              </group>
            </group>
```

Reply:
Please do this structure here and elsewhere in the style. Use the review-of term is better than hard coding English text

Reply:
Noted. Thanks for the pointer to review-of. I didn't have that term in the copy of the CSL specification I was working from.